doc/Getting-started/index.md

```diff
@@ -12,40 +11,26 @@ sort_idx: 10
 * TOC
 {:toc}
 
-This section provides guidance on how to set up Sandbox and run your first Kaa application that will work with the Kaa platform.
 
-# Kaa Sandbox
+*Kaa Sandbox* is a pre-configured virtual environment specifically designed for speeding up the setup of your private Kaa platform instance for educational, development, and proof-of-concept purposes.
+The Sandbox also includes a selection of demo applications that illustrate various aspects of the platform functionality.
 
-Kaa Sandbox is a private Kaa environment which includes demo client applications. 
-Sandbox includes all necessary Kaa components in a convenient virtual environment that can be set up in just 5 minutes! 
-With the use of Kaa Sandbox, anyone can learn Kaa, build a proof of concept and test their own applications locally.
-
-## Video tutorial
+This page provides guidance on how to set up Sandbox and run your first Kaa application that will work with the Kaa platform.
```

Comment:
Chicken and egg problem : While *Kaa Sandbox* definition is required for explaining the purpose of this page, it's unclear why *Kaa Sandbox* definition is required without a purpose of this page.